src/Util/Cpu.php

```diff
@@ -46,7 +46,7 @@ public function getThreadsCount(): int
     {
         try {
             $result = $this->shell->execute('nproc');
-            $threadCount = (int)$result->getOutput() ?? 1;
+            $threadCount = max((int)$result->getOutput(), 1);
```

Comment:
What was wrong with that code?

Reply:
http://sandbox.onlinephpfunctions.com/code/398cb3694216b011d30600973ecba2ad34cf8cc3